pretext/Lists/ListMethods.ptx

```diff
@@ -188,23 +155,89 @@ print(mylist)
       </row>
     </tabular>
   </table>
-  <p>Details for these and others
-            can be found in the <url href="http://docs.python.org/py3k/library/stdtypes.html#sequence-types-str-bytes-bytearray-list-tuple-range" visual="http://docs.python.org/py3k/library/stdtypes.html#sequence-types-str-bytes-bytearray-list-tuple-range">Python Documentation</url>.</p>
-  <p>It is important to remember that methods like <c>append</c>, <c>sort</c>,
-            and <c>reverse</c> all return <c>None</c>.  This means that re-assigning <c>mylist</c> to the result of sorting <c>mylist</c> will result in losing the entire list.  Calls like these will likely never appear as part of an assignment statement (see line 8 below).</p>
-  <program xml:id="ch4_sec10_p2" interactive="activecode" language="python">
+
+
+
+  <p>Here are some examples on Be sure to experiment with these methods to gain a better understanding of what they do. You are expected to be comfortable using these methods!</p>
```

Comment:
Looks like you are missing the rest of the sentence here

Reply:
Good catch! thanks.